C#/Botkit/Microsoft.BotKit.Adapters.Slack/SlackBotWorker.cs

```diff
@@ -1,14 +1,22 @@
 // Copyright(c) Microsoft Corporation.All rights reserved.
 // Licensed under the MIT License.
 
-using Microsoft.BotKit;
 using Microsoft.BotKit.Core;
 using System.Threading.Tasks;
+using SlackAPI;
+using Microsoft.Bot.Schema;
+using System;
+using Newtonsoft.Json;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
 
 namespace Microsoft.BotKit.Adapters.Slack
 {
     public class SlackBotWorker : BotWorker
     {
+        public SlackClient api;
+        readonly Botkit controller;
```

Comment:
The original inherits this from its base class'. While this is private in BotWorker.cs, could we change it to protected / public and inherit from there?

Reply:
We are not using it, we access the controller with GetController() so I will remove it.